paper.md

```diff
@@ -1,36 +1,51 @@
 ---
-title: 'PyGBe: Python, GPUs and Boundary elements for biomolecular electrostatics'
+title: 'PyGBe: Python and GPU Boundary-integral solver for electrostatics'
 tags:
   - electrostatics
   - biophysics
   - Poisson-Boltzmann
```

Comment:
How do tags work here? Is this a separate JOSS paper, or an update? This because, in the context of LSPR, Poisson-Boltzmann is not used